container_service_extension/mqtt_extension_manager.py

```diff
@@ -398,6 +402,7 @@ def delete_extension_token(self, token_id):
         """Delete the specified token with token_id.
 
         :param str token_id: the token's id
+
```

Comment:
Whitespace change... not sure if intentional.

Reply:
Thank you, Chintan. I just added it to try to be consistent with container_service_extension/compute_policy_manager.py, which separate the :param section from the :raise/:return section of the comments.